### PR TITLE
Fixed `-l` flag for finding live links.

### PR DESCRIPTION
**Pull request by KingAkeem**
Takes list of URLs and asychrnously calls HEAD request on each
link in the list and test status code for a 200 response. If the
response is not 200 or takes longer than 8 seconds, the link is
declared dead. Also switched from urllib.request to requests for
not only simplicity but thread-safety also.

Fixes: #61

### CODE REVIEW

**Comment by PSNAppz** [comment]
@KingAkeem Fix ext flag

**Comment by KingAkeem** [comment]
What about issue #58 ? I was going to work on it but I didn't know if someone was already working on it.

File: modules/getweblinks.py [hunk 1]
```diff
@@ -56,25 +65,29 @@ def getLinks(soup):
             soup: BeautifulSoup instance currently being used.
 
         Returns:
-            websites: List of websites that were found
+            wfebsites: List of websites that were found
```
[the remark this answers is not in the document]
Comment (KingAkeem) [reply modules/getweblinks.py]: fixed

**Comment by PSNAppz** [comment]
@KingAkeem The test seems broken. 

`ImportError: No module named 'requests'`

Any idea ? The requests library is added to requirements btw.

**Pull request merged**